src/connection-manager/data/amendments_enabled.json

```diff
@@ -309,112 +309,6 @@
     }
     ]
   },
-  {"networks": "dev",
```

Comment:
Why was this removed?

Reply:
This is because of the reset. I think it should not be here in the first place. I will fix on previous PR and merge